WordPress/WordPressScreenshotGeneration/WordPressScreenshotGeneration.swift

```diff
@@ -55,65 +49,65 @@ class WordPressScreenshotGeneration: XCTestCase {
         }
         postEditorScreenshot.close()
 
-        // Get a screenshot of the editor with keyboard (iPad only)
-        if XCTestCase.isPad {
-            let ipadScreenshot = try MySiteScreen()
-                .showSiteSwitcher()
-                .switchToSite(withTitle: "weekendbakesblog.wordpress.com")
-                .goToPostsScreen()
-                .showOnly(.drafts)
-                .selectPost(withSlug: "easy-blueberry-muffins")
-            try BlockEditorScreen().selectBlock(containingText: "Ingredients")
-            sleep(imagesWaitTime) // wait for post images to load
-            try BlockEditorScreen().thenTakeScreenshot(7, named: "Editor-With-Keyboard")
-            ipadScreenshot.close()
-        } else {
-            postList.pop()
-        }
-
-        // Get My Site screenshot
-        let mySite = try MySiteScreen()
-            .showSiteSwitcher()
-            .switchToSite(withTitle: "tricountyrealestate.wordpress.com")
-            .thenTakeScreenshot(4, named: "MySite")
-
-        // Get Media screenshot
-        _ = try mySite.goToMediaScreen()
-        sleep(imagesWaitTime) // wait for post images to load
-        mySite.thenTakeScreenshot(6, named: "Media")
-
-        if !XCTestCase.isPad {
-            postList.pop()
-        }
-
-        // Get Stats screenshot
-        let statsScreen = try mySite.goToStatsScreen()
-        statsScreen
-            .dismissCustomizeInsightsNotice()
-            .switchTo(mode: "months")
-            .thenTakeScreenshot(3, named: "Stats")
-
-        // Get Discover screenshot
-        // Currently, the view includes the "You Might Like" section
-        try TabNavComponent()
-            .goToReaderScreen()
-            .openDiscoverTab()
-            .thenTakeScreenshot(2, named: "Discover")
-
-        // Get Notifications screenshot
-        let notificationList = try TabNavComponent()
-            .goToNotificationsScreen()
-        if XCTestCase.isPad {
-            notificationList
-                .openNotification(withSubstring: "commented on")
-        }
-        notificationList.thenTakeScreenshot(5, named: "Notifications")
+//        // Get a screenshot of the editor with keyboard (iPad only)
```

Comment:
I haven't updated the screenshot Generation for the WordPress app yet. I only focused on Jetpack.